_usage/command-line.md

```diff
@@ -1,212 +1,121 @@
-                            OpenSSL: no
-                             GnuTLS: yes
-                               MARS: yes
+### Using `tsql` to connect
 
-```
+`tsql` is a command line tool that is shipped as part of FreeTDS. It allows you
+to connect and to interact with Babelfish (or any other TDS enabled data source)
+from your Linux terminal.
 
-Now that the syntax and the client settings are clear, we can easily connect to the server:
+Here is an example how to connect to Babelfish with `tsql`:
 
 ```bash
 tsql -S database.example.com -p 1433 -U postgres -P secretpassword
 ```
 
-A word of caution: The tsql binary allows you to send
-the password as part of the command. This normally results in a security
-problem. Fortunately, tsql replaces the command name so that the password is not
-visible in the process table, as shown in the next listing:
-
-```bash
- 425705 pts/5    Ss     0:00  |   \_ bash
- 426036 pts/5    S+     0:00  |   |   \_ tsql -H database.example..com -p 1433 -U postgres -P ************* -D postgres
-```
-
-If your connection has been established successfully, you should find yourself in
-a command line:
-
-```bash
-1> SELECT 1+1
-2> GO
-
-2
-(1 row affected)
-
-1> SELECT 1+1 AS x UNION ALL SELECT 2+2
-2> GO
-x
-2
-4
-(2 rows affected)
-```
-
-There are a couple of things worth mentioning here: First of all, we have to use
-<code>GO</code> to make Babelfish execute the command. This is just like in standard MS SQL,
-as well as in Sybase. But there is more, which might be alien to users
-experienced with PostgreSQL: <code>SELECT 1+1</code> does not yield a column title, so to
-ensure that a title is provided, we need to alias the column. You need to be aware
-of such small details in order to ensure that the code written stays portable, and still
-works with Microsoft SQL Server as well as PostgreSQL.
-
+Like in `sqlcmd`, the command `USE` selects a database, and `GO` is used to send
+a batch of SQL statements to the server.
 
-### Using psql to connect
+Note that `tsql` is not officially supported.
 
-You can also use PostgreSQL's `psql` to connect to Babelfish. However, this is
-**strongly** discouraged, and the behavior will be unpredictable. It is
-therefore not recommended to to that &ndash; in fact it can be pretty dangerous.
 
-Babelfish comes with a modified version of `psql`, which allows to set
-the SQL dialect:
+### Using `psql` to connect
 
-```bash
-SET babelfishpg_tsql.sql_dialect = 'tsql';
-\tsql on
-
-INSERT INTO some_table VALUES (1);
-GO
-```
+You can also use PostgreSQL's `psql` to connect to Babelfish.  `psql` has to
+connect to the PostgreSQL port (by default 5432), and you will have to use
+PostgreSQL SQL syntax instead of T-SQL.  The only thing that you should avoid
+is changing the SQL dialect with the
+[`babelfishpg_tsql.sql_dialect`](/docs/internals/configuration/#babelfishpg_tsqlsql_dialect)
+configuration parameter, but that recommendation is not specific to `psql`.
```

Comment:
I'd leave lines 111 to 116 out, since I've mentioned it earlier as connecting on the PG port, and using the native PG dialect.

Reply:
I think we should leave that.
This is a different issue: you can connect on the PostgreSQL port and change the configuration parameter `babelfishpg_tsql.sql_dialect`. That's something you shouldn't do.

Reply:
Wouldn't that content belong in the section about configuration parameters (since it's not specific to psql)?  There are a lot of things you shouldn't do with psql (or any client)...

Reply:
You are right. I moved it there in d69f9160410b00b1a60a539271eba95b2047bf22.

I left the rest of the description, but added a code sample for a `psql` connection out of symmetry with the other sections.